protocol-designer/src/components/StepEditForm/fields/TipPositionField/TipPositionInput.module.css

```
@@ -65,3 +65,14 @@
   position: relative;
   left: 9px;
 }
+
+.tip_position_icon {
+  height: 1.5rem;
+  width: 1.5rem;
+  cursor: pointer;
+  color: #24313f;
```

nit
Adding color info as a comment might be useful for the next gen?

```suggestion
  color: #24313f;  /* black80 */
```